mllib/src/test/scala/org/apache/spark/ml/source/image/ImageFileFormatSuite.scala

```diff
@@ -96,14 +95,14 @@ class ImageFileFormatSuite extends SparkFunSuite with MLlibTestSparkContext {
       .collect()
 
     assert(Set(result: _*) === Set(
-      Row("29.5.a_b_EGDP022204.jpg", "kittens", Date.valueOf("2018-01-01")),
```

Comment:
I reverted changes made by https://github.com/apache/spark/pull/33709/files#r688851936. Now the test looks the same as in `branch-3.2`.